README.md

```diff
@@ -1,2 +1,91 @@
-# curveadm
+# CurveAdm
 Deploy and manage CurveFS cluster
+
+Table of Contents
+===
+
+* [Requirement](#requirement)
+* [Installation](#installation)
+* [Usage](#usage)
+    * [Deploy Cluster](#deploy-cluster) 
+    * [Mount FileSystem](#mount-filesystem)
+    * [Umount FileSystem](#umount-filesystem)
+* [Devops](#devops)
+
+Requirement
+---
+
+CurveAdm depends on docker, please [install docker](https://docs.docker.com/engine/install/) first.
+
+Please make sure the docker daemon has running, you can run the below command to verify:
+
+```shell
+$ sudo docker run hello-world
+```
+
+Installation
+---
+
+```shell
+$ sh -c "$(curl -fsSL https://raw.githubusercontent.com/opencurve/curveadm/develop/scripts/install.sh)"
+```
+
+Usage
+---
+
+### Deploy Cluster
+
+Prepare cluster topology, you can refer to the sample configuration:
+
+* [cluster](examples/cluster/services.yaml)
+* [stand-alone](examples/stand-alone/services.yaml)
```

Comment:
targets in cluster mode and stand-alone mode are the same 3, stand-alone mode should be 1?

Reply:
> targets in cluster mode and stand-alone mode are the same 3, stand-alone mode should be 1?

fixed.